moped-editor/src/views/projects/projectView/ProjectTeamTable.js

```diff
@@ -244,164 +242,30 @@ const ProjectTeamTable = ({
             multiline
             inputProps={{ maxLength: 125 }}
             value={val && val !== "null" ? val : ""}
-            onChange={e => props.onChange(e.target.value)}
+            onChange={(e) => props.onChange(e.target.value)}
           />
         );
       },
     },
   ];
 
-  /**
-   * Data handlers for editable actions based on isNewProject boolean <MaterialTable>
-   */
-  const isNewProjectActions = {
-    true: {
```

Comment:
these are functions that handled adding personnel when adding a new project. Which doesn't happen anymore.